fields/types/boolean/test/explorer.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
 	Field: require('../BooleanField'),
 	Filter: require('../BooleanFilter'),
+	readme: require('fs').readFileSync('./fields/types/boolean/Readme.md', 'utf8'),
```

Comment:
I understand this uses `brfs` to inline the code, but this way of doing it makes the dependency on the readme file opaque (and relies heavily on browserify if I understand correctly).

Why not import the file and define a loader that reads the `.md` files as a string?